File: helloworld.cs
/**
 * CC 
 * A simple program in C# 
 * With the help of Sololearn
 **/
using System;

namespace HelloWorldProgram{
 
 class HelloProgram{
     
     static void Main(string[] args){
         const var hellowWorldStatement = "Hello World!"; 
         console.WriteLine(hellowWorldStatement);

        if (hellowWorldStatement == 10){
            console.WriteLine("It's equal to the integer 10!!");
        }else{
            console.WriteLine("You have failed to solve it's equivalency.. :(");
        }
        boolean endLoop = false; 
        while(!endLoop){
            console.WriteLine("Going to end that loop..")
            endLoop = true; 
        }
        return 0; 
     }    
 }
 
}

File: helloworld.cpp
/**
 * CC 
 * A Hello World program in C++ 
 * With the help of Sololearn
 **/
#include <iostream>
#include <cstdlib>
using namespace std; 

int main{

    const string helloWorldStatement = "Hello World!"; 
    int i = 10;
    int randomNumber = rand() % 15;
    cout<<helloWorldStatement << endl;
    while(i != 0){
        cout<<"**" << helloWorldStatement << "**" << endl; 
        i--;
        for(int j = 0; j < 5; j++){
            cout<<"Wowza You hit the for loop!: "<< j << endl;
        }
        do{
            cout<<"Oh my hit the do while loop: "<< randomNumber << endl;
            randomNumber--; 
        }while(randomNumber > 0)
    }


    return 0;

}
